scss/custom/includes/components/_button.scss

```diff
@@ -38,3 +38,16 @@
 .button-long {
   @include button-colors($white, $cfa-red, $white);
 }
+
+.join-button {
+  @include button-colors($cfa-red, $white, $white);
+  border-radius: 2px;
+  cursor: pointer;
+  font-family: $font-family-alt;
+  font-size: 0.875rem;
+  font-weight: 600;
+  border: 7px 17px 7px 17px;
```

Comment:
No need for this line. Check out this link for border syntax: https://developer.mozilla.org/en-US/docs/Web/CSS/border

But we don't even need it here I don't think.